README.md

```diff
@@ -21,34 +21,35 @@ thus this repository is very much a work in progress.
 # See `otk --help` for help.
 ```
 
-## Pre Commit Checks
+## Documentation
+
+There is documentation about the [format](./doc/format.md) and the available
+[directives](./doc/directives.md).
+
+## Examples
+
+Read the [examples](./example) to see what omnifests look like.
+
+## Development
+
+### Pre Commit Checks
 
-To check your code for basic problems, we use https://pre-commit.com
+To check our code for basic problems we use [pre-commit](https://pre-commit.com)
 The tool itself will be installed by the `pip` command above (see [Usage](#Usage)) after that you
 should run
 
 ```shell
 pre-commit install
 ```
 
-After that the system automatically checks upon commit, or you can run it against the whole
+After this the system automatically checks upon commit, or you can run it against the whole
 repository including all the tests with:
 
 ```
-make test
+make lint
```

Comment:
"including all the tests" is not really true after this change

Reply:
Changed it back to `make test` which now includes `pre-commit` again as per #28.